schemas/snuba-uptime-results.v1.schema.json

```diff
@@ -27,6 +27,10 @@
     "environment": {
       "type": "string",
       "description": "The environment associated with this check"
+    },
+    "in_incident": {
+      "type": "boolean",
+      "description": "Indicates this results is part of a detected incident"
```

Comment:
Wondering if it makes sense for this to be an int so that we have flexibility if we want to label things differently later?